docs/fundamentals/gateway/custom-gateway.md

```diff
@@ -0,0 +1,326 @@
+(custom-gateway)=
+# Customize the Gateway
+A Jina Gateway is customizable and can be implemented in a similar way Executors are implemented.
+With Custom Gateways, Jina gives power back to the users, by allowing to implement any server, protocol and interface on
```

Comment:
"...by allowing _them_ to implement any server, protocol and interface _at_ the gateway level."